docs/_articles/en/vscode-desktop/java-setup.md

```diff
@@ -73,54 +30,79 @@ Zulu Java builds are TCK-tested and free to download and use without restriction
 7. Download the installer.
 8. Once the file is downloaded open it and complete the installation steps.
 
-Inside Visual Studio Code, set `salesforcedx-vscode-apex.java.home` to one of the following values.
+### Oracle Java 17 (Officially Tested)
+
+1. Navigate to the [download page](https://www.oracle.com/java/technologies/downloads/).
+2. Navigate to **Java SE Development Kit 17.0.2 downloads**.
```

Comment:
Looks like the latest version of Java 17 is 17.0.10, when I click on the Java 17 tab from the link in Step 1.

```suggestion
2. Navigate to **Java SE Development Kit 17.0.10 downloads**.
```